test/bdd/features/vc_e2e_api.feature

```diff
@@ -21,12 +21,17 @@ Feature: Using VC REST API
+      | myprofilewithdidelem    | university_degree.json       | did:elem:EiAWdU2yih6NA2IGnLsDhkErZ8aQX6b8yKt7jHMi-ttFdQ          | 5AcDTQT7Cdg1gBvz8PQpnH3xEbLCE1VQxAJV5NjVHvNjsZSfn4NaLZ77mapoi4QwZeBhcAA7MQzaFYkzJLfGjNnR | JWS             |                                                                                                                                                               | did:elem        |    Ed25519Signature2018 |           |
+      | myprofilewithdidsov     | university_degree.json       |                                                                  |                                                                                          | JWS             |       {"driverURL":"https://uniregistrar.io/1.0/register?driver-universalregistrar/driver-did-sov","options": {"network":"danube"}}                           | did:sov:danube  |    Ed25519Signature2018 |           |
+      | myprofile_ud_ed25519    | university_degree.json       |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  Ed25519  |
+      | myprofile_ud_p256_jws   | university_degree.json       |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  P256     |
+      | myprofile_ud_p256_pv    | university_degree.json       |                                                                  |                                                                                          | ProofValue      |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  P256     |
+      | myprofile_prc_ed25519   | permanent_resident_card.json |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  Ed25519  |
```

Comment:
this profile like myprofile_prc

Reply:
That't with uniRegistrar value. It's empty in this profile.